src/main/java/dao/BandDao.java

```diff
@@ -0,0 +1,33 @@
+package dao;
+
+import entity.Band;
+
+import java.util.List;
+
+/**
+ * Created by ������� on 12.03.2017.
+ */
+public interface BandDao {
+
```

Comment:
Here findById(id), findAll(), save(band) and delete(Integer id) will be enough for now